internal/db/filter/parser.go

```diff
@@ -265,11 +272,72 @@ func convertValue(val *Value) interface{} {
 		return strings.ToLower(*val.Boolean) == "true"
 	}
 
+	if val.ValueList != nil {
+		// Convert list of values to slice
+		var values []interface{}
```

Comment:
We should use `any` for new code:

```suggestion
		var values []any
```